state/movie/operations/normalize/movie.js

```diff
@@ -0,0 +1,11 @@
+import { normalize, schema } from 'normalizr';
+
+const normalizeMovie = (data) => {
```

Comment:
Please, put this file near normalizeMovies function which you have created for other operations

Reply:
Done